src/Refactoring-Core/RBPullUpMethodRefactoring.class.st

```diff
@@ -142,66 +149,68 @@ RBPullUpMethodRefactoring >> checkSuperSendsFromPushedUpMethods [
 	| siblings |
-	siblings := class superclass subclasses reject: [:each | each = class].
+	siblings := superClass subclasses reject: [:each | each = class].
 	siblings do: [:aRBClass | self checkSiblingSuperSendsFrom: aRBClass]
 ]
 
 { #category : #preconditions }
 RBPullUpMethodRefactoring >> checkSuperclass [
 	| overrideSelectors |
 	overrideSelectors := selectors
-		select: [ :each | class superclass directlyDefinesMethod: each ].
+		select: [ :each | superClass directlyDefinesMethod: each ].
 	overrideSelectors := overrideSelectors
 		reject: [ :each | 
 			| myTree superTree |
 			myTree := class parseTreeFor: each.
-			superTree := class superclass parseTreeFor: each.
+			superTree := superClass parseTreeFor: each.
 			superTree equalTo: myTree exceptForVariables: #() ].
 	overrideSelectors isEmpty
 		ifTrue: [ ^ self ].
-	class superclass isAbstract
+	superClass isAbstract
 		ifFalse: [ self
 				refactoringError:
 					('Non-abstract class <2p> already defines <1p>'
 						expandMacrosWith: overrideSelectors asArray first
-						with: class superclass) ].
+						with: superClass) ].
 	overrideSelectors do: [ :each | self checkBackReferencesTo: each ]
 ]
 
-{ #category : #private }
-RBPullUpMethodRefactoring >> copyDownMethod: aSelector [
+{ #category : #initialization }
+RBPullUpMethodRefactoring >> class1 [
+	^ class
+]
 
+{ #category : #private }
+RBPullUpMethodRefactoring >> copyDownMethod: aSelector [ 
 	| oldProtocol oldSource superclassDefiner subclasses refactoring |
-
-	superclassDefiner := class superclass whoDefinesMethod: aSelector.
-	superclassDefiner ifNil: [ ^ self ].
+	superclassDefiner := superClass whoDefinesMethod: aSelector.
+	superclassDefiner ifNil: [^ self ].
 	oldSource := superclassDefiner sourceCodeFor: aSelector.
-	oldSource
-		ifNil: [ self
-				refactoringFailure: ( 'Source code for <1s> superclass method not available' expandMacrosWith: aSelector )
-			].
+	oldSource ifNil: 
+			[ self 
+				refactoringFailure: ('Source code for <1s> superclass method not available' 
+						expandMacrosWith: aSelector)].
 	oldProtocol := superclassDefiner protocolsFor: aSelector.
-	subclasses := class superclass subclasses reject: [ :each | each directlyDefinesMethod: aSelector ].
-	subclasses isEmpty
-		ifTrue: [ ^ self ].
-	( ( superclassDefiner parseTreeFor: aSelector ) superMessages
-		detect: [ :each | superclassDefiner directlyDefinesMethod: each ]
-		ifNone: [ nil ] ) notNil
-		ifTrue: [ self
-				refactoringError:
-					( 'Cannot pull up <1s> since we must copy down the superclass method in <2p><n>to the other subclasses, and the superclass method sends a super message which is overriden.'
+	subclasses := superClass subclasses 
+				reject: [:each | each directlyDefinesMethod: aSelector].
+	subclasses isEmpty ifTrue: [^ self ].
+	((superclassDefiner parseTreeFor: aSelector) superMessages 
+		detect: [:each | superclassDefiner directlyDefinesMethod: each]
+		ifNone: [nil]) ifNotNil: 
```

Comment:
Same as before. You did not introduce it but we can maybe take it as an opportunity to clean the code.